### PR TITLE
Ensure we're using thread of the newly created message

### LINKED ISSUE #16: Same repo notifications in different threads
As we've seen with the `Bootstrap` repo, the bot is currently sending notification messages to different threads.

As per our initial requirements, the bot should send messages about the same repo to the same thread.

### PR DESCRIPTION
**Pull request by yuri-sergiichuk**
This PR fixes #16.

The bug was introduced as part of the codebase refactoring done in 67a22722e2451bcb9d54256036e5a30ce6447c6e where we started using the message to be sent instead of the already sent message while building up the response.

### CODE REVIEW

**Comment by yuri-sergiichuk** [comment]
@armiol @alexander-yevsyukov PTAL

File: gradlew [hunk 1]
```diff
@@ -1,19 +1,23 @@
 #!/usr/bin/env sh
 
 #
-# Copyright 2015 the original author or authors.
+# Copyright 2020, TeamDev. All rights reserved.
```
Comment (armiol) [inline gradlew:4]: Not sure why changing this copyright header.

**Review by armiol — APPROVED** [review]
@yuri-sergiichuk LGTM with a comment.

File: gradlew (hunk 1 above)
Comment (yuri-sergiichuk) [reply gradlew]: IDEA changes automatically while upgrading the Gradle version. Reverted.

**Pull request merged**